static/js/tools/map/chart_loader.tsx

```diff
@@ -739,8 +678,26 @@ function loadChartData(
     return;
   }
   const calculateRatio = statVar.perCapita && statVar.denom ? true : false;
-  // populate mapValues with data value for each geo that we have geoJson data for.
-  for (const geoFeature of rawData.geoJsonData.features) {
+  // populate mapValues with data value for each geo that we have geoJson data
+  // for.
+  if (!geoJsonData) {
```

Comment:
return early here is crucial

Reply:
could you add the note in code too (along with why it's crucial for future code updates)?

Reply:
This is moved into a separate useEffect and I added a note there. The statement management will need more cleanup as we proceed with the refactor